### PR TITLE
chore: Update config for new software

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

This PR updates the config to prepare for the new gnatss version.

### CODE REVIEW

**Comment by lsetiawan** [comment]
@johnbdesanto Please checkout this branch locally, update the nov files and run the posfilter. After that, please send me your gps solution csv file :) Thanks!

File: config.yaml [hunk 1]
```diff
@@ -1,45 +1,67 @@
...
+posfilter:
+  export:
+    full: false
+  atd_offsets:
+    forward: 0.0053
+    rightward: 0
+    downward: 0.92813
+  input_files:
+    novatel:
+      path: path/to/NCL1_INSPVAA.dat
```
Comment (johnbdesanto) [inline config.yaml]: Replace NCL1 -> NDP1

File: config.yaml [hunk 2]
```diff
@@ -1,45 +1,67 @@
...
+    novatel_std:
+      path: path/to/NCL1_INSSTDEVA.dat
```
Comment (johnbdesanto) [inline config.yaml]: Replace NCL1 -> NDP1

**Review by johnbdesanto — APPROVED** [review]
lgtm

**Pull request merged**